oss-cloud-frontend/src/components/ContributionsPage.vue

```diff
@@ -35,21 +56,21 @@
         <div v-else>{{ item.status }}</div>
       </template>
 
-      <template v-slot:item.dateCreated="{ item }">
-        {{ new Date(item.dateCreated).toJSON().slice(0,10) }}
-      </template>
+      <template
+        v-slot:item.dateCreated="{ item }"
+      >{{ new Date(item.dateCreated).toJSON().slice(0,10) }}</template>

```

Comment:
Let's using something else to format the date. maybe moment.js

Reply:
Since that is my code, I would like to fix it on my current branch (which is related to this)